x-pack/metricbeat/module/containerd/blkio/_meta/docs.asciidoc

```diff
@@ -0,0 +1 @@
+This is the blkio metricset of the module containerd.
```

Comment:
This docs page looks quite basic. Maybe we can cover the rational of how metrics and pcts are calculated here as suggested at https://github.com/elastic/beats/pull/29247#discussion_r761951922? The better we do our docs the easier it would be to provide support in the future.